playbooks/hue_server_restart.yml

```
@@ -0,0 +1,14 @@
+# Copyright 2022 TOSIT.IO
+# SPDX-License-Identifier: Apache-2.0
+
+---
+- name: Start Hue Server
```

```suggestion
- name: Restart Hue Server
```
Same line 10 bellow.

---

Done